modules/btcpayserver.nix

```diff
@@ -101,25 +101,30 @@ let
   nbLib = config.nix-bitcoin.lib;
   nbPkgs = config.nix-bitcoin.pkgs;
 
-  bitcoind = config.services.bitcoind;
+  inherit (config.services) bitcoind liquidd;
 in {
   inherit options;
 
   config = mkIf cfg.btcpayserver.enable {
-    services.bitcoind.enable = true;
+    services.bitcoind = {
+      enable = true;
+      listen = true;
+      rpc.users.btcpayserver = {
+        passwordHMACFromFile = true;
+        rpcwhitelist = cfg.bitcoind.rpc.users.public.rpcwhitelist ++ [
+          "setban"
+          "generatetoaddress"
+          "getpeerinfo"
+        ];
+      };
+      extraConfig = ''
+        whitelist=${nbLib.address cfg.nbxplorer.address}
```

Comment:
Should also be whitelisted in [liquidd](/https://github.com/fort-nix/nixbitcoin.org/blob/master/configuration.nix#L58) (same problem)